simplified-ui-catalog/src/main/res/layout/feed_lane_item.xml

```diff
@@ -1,25 +1,27 @@
 <?xml version="1.0" encoding="utf-8"?>
-<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
-  xmlns:app="http://schemas.android.com/apk/res-auto"
-  android:layout_width="wrap_content"
-  android:layout_height="wrap_content"
-  android:clipChildren="false"
-  android:clipToPadding="false"
-  android:orientation="vertical">
 
-  <androidx.cardview.widget.CardView
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Since there's just a CardView, do we need a parent view? If so, maybe we should use a FrameLayout instead then

Reply:
I freely admit I have no idea. I've not looked at any of the UI code for years, and even then, I wasn't the person who wrote it.

What arrangement of layouts were you thinking of?

```
<FrameLayout>
  <CardView>
    <ImageView/>
  </CardView>
</FrameLayout>
```

Like that?

Reply:
Removing the parent view completely seems to work fine!

Reply:
Cool! But for the record, I was thinking about this (like you said), if we were not to remove the parent view:

```
<FrameLayout>
  <CardView>
    <ImageView/>
  </CardView>
</FrameLayout>
```